WordPress/Classes/ViewRelated/Post/WPPostViewController.m

```diff
@@ -1572,7 +1572,17 @@ - (BOOL)hasFailedMedia
 
 - (BOOL)isMediaUploading
 {
-    return self.mediaInProgress.count > 0;
+    if (self.mediaInProgress.count == 0){
+        return NO;
+    }
+    
+    __block BOOL mediaUploading = NO;
```

Comment:
Also performs faster - looping with `enumerateKeysAndObjectsUsingBlock` only makes sense if you need the extra parameters.